examples/apps/cra-demo/src/App.js

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import TinyliciousClient from "@fluid-experimental/tinylicious-client";
+import { SharedMap } from "@fluidframework/map";
+
+const getContainerId = () => {
+    let isNew = false;
+    if (window.location.hash.length === 0) {
+        isNew = true;
+        window.location.hash = Date.now().toString();
+    }
+    const id = window.location.hash.substring(1);
+    return { id, isNew };
+};
+
+TinyliciousClient.init();
+
+const getFluidData = async () => {
```

Comment:
I'm not sure I'm a huge fan of this function being pulled out. It makes the app code a bit more condensed but also requires you to use a .then and pass back the initialObjects.

More a nit since it's just preference.

Reply:
@sumedhb1995 and I went back and forth on this one a bit. I really like that we can define a function that simply returns the Fluid data. It tells a very easy to understand story. I don't __love__ that it has to use a promise like that either, but I feel its easier to understand than creating another async/await function that I then need to call. 

Happy to take feedback on it and continue to iterate over time.